inst/resources/activate.R

```diff
@@ -96,133 +115,132 @@ local({
 
     }
 
-    # load the project
-    return(renv::load())
+    return(load_project())
 
   }
 
-  # try to bootstrap an renv installation   
+  # try to bootstrap an renv installation
   bootstrap <- function(version, library) {
-  
```

Comment:
Please configure your editor to avoid overaggressive whitespace changes (as they make the GitHub diffs noisier)